activerecord/test/cases/associations/has_many_associations_test.rb

```diff
@@ -1000,7 +1000,7 @@ def test_select_query_method
   end
 
   def test_select_with_block
-    assert_equal [1], posts(:welcome).comments.select { |c| c.id == 1 }.map(&:id)
+    assert_equal [1], posts(:welcome).comments.filter_map { |c| c.id if c.id == 1 }
```

Comment:
This might be false negative for the `Performance/SelectMap` cop.
I'd prefer to improve the cop rather than rewrite them one by one by hand.

cc @koic

Reply:
This is because the following configuration in .rubocop.yml. So it is not `Performance/SelectMap` cop' false negative. 

```yaml
Performance:
  Exclude:
    - '**/test/**/*'
```

https://github.com/rails/rails/blob/v6.1.3.1/.rubocop.yml#L20-L22

Reply:
@kamipo Do I need to add this to rubocop yml 
` Performance/SelectMap:
  Enabled: true
  Exclude:
    - '**/test/**/*'`

Reply:
I think `Include: ['**/test/**/*']` does not work as expected. Probably the behavior that `exclue: []` expects.

```diff
 Performance/SelectMap:
   Enabled: true
+  Exclude: []
```

Reply:
Okay Thanks I will make the required changes.

Reply:
And I think the maintainers will decide what to do with this configuration.

Reply:
I will make same chages to MapCompact too.

Reply:
Please note the name of the test method being modified here.

Reply:
Good point. `select` with block should be used here in the `test_select_with_block`.